Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```
@@ -894,6 +894,95 @@ script:
     - contextPath: Cylance.Device.Deleted
       description: A boolean to check if the device was deleted.
       type: boolean
+  - name: cylance-optics-create-instaquery
+    arguments:
+    - name: name
+      description: |
+        InstaQuery name
+        ok test
+      required: true
+    - name: description
+      description: InstaQuery description
+      required: true
+    - name: artifact
+      description: InstaQuery artifact, select from the list
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - File
+      - Process
+      - NetworkConnection
+      - RegistryKey
+    - name: match_value_type
+      description: InstaQuery value type to match, select from the list
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - File.Path
+      - File.Md5
+      - File.Sha2
+      - File.Owner
+      - File.CreationDateTime
+      - Process.Name
+      - Process.Commandline
+      - Process.PrimaryImagePath
+      - Process.PrimaryImageMd5
+      - Process.StartDateTime
+      - NetworkConnection.DestAddr
+      - NetworkConnection.DestPort
+      - RegistryKey.ProcessName
+      - RegistryKey.ProcessPrimaryImagePath
+      - RegistryKey.ValueName
+      - RegistryKey.FilePath
+      - RegistryKey.FileMd5
+      - RegistryKey.IsPersistencePoint
+    - name: match_values
+      description: Value to search in InstaQuery
+      required: true
+    - name: zone
+      description: Zone of the object
+      required: true
+    - name: match_type
+      description: Match type fuzzy or exact
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - Fuzzy
+      - Exact
+    description: Create a cylance InstaQuery
+    outputs:
+    - contextPath: InstaQuery.New.id
+      description: The unique identifier of the created InstaQuery.
+      type: string
+    - contextPath: InstaQuery.New.created_at
+      description: The Date and Time that the InstaQuery was created.
+      type: date
+    - contextPath: InstaQuery.New.progress
+      description: The progress of the InstaQuery.
+      type: string
+  - name: cylance-optics-get-instaquery-result
+    arguments:
+    - name: query_id
+      description: InstaQuery ID
+      required: true
+    description: Get a cylance InstaQuery search result
+    outputs:
+    - contextPath: InstaQuery.Results.result
+      description: The InstaQuery results.
+      type: string
+  - name: cylance-optics-list-instaquery
+    arguments:
+    - name: page
+      description: number of page to collect
```

```suggestion
    - name: page
      description: The page number to collect. If not specified, the default is 1.
```
Our standard is to name this `page_number`